memoized_lcnaf.json

```diff
@@ -347,5 +347,14 @@
   "Singapore Subject": "https://id.loc.gov/authorities/names/n79059023",
   "Singapore": "https://id.loc.gov/authorities/names/n79059023",
   "United States. Army.": "https://id.loc.gov/authorities/names/n79063202",
-  "United States. Army": "https://id.loc.gov/authorities/names/n79063202"
+  "United States. Army": "https://id.loc.gov/authorities/names/n79063202",
+  "New York (State)": "https://id.loc.gov/authorities/names/n80126293",
+  "Belarus": "https://id.loc.gov/authorities/names/n91128701.html",
```

Comment:
Let's remove the `.html` suffix. The no suffix form redirects to the .html one. The no suffix form is the correct identifier form we use.